vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher_list_reader.cc

```diff
@@ -41,6 +42,17 @@ PublisherListReader::ParseError PublisherListReader::Parse(
       uncompressed = std::move(*message.mutable_prefixes());
       break;
     }
+    case PublisherList::BROTLI_COMPRESSION: {
```

Comment:
Brotli gives better results that gzip for the sorted prefix list, and is easy to support since we are already using it for the channel response payload. In the future, we may introduce more specialized compression algorithms similar to how Safe Browsing uses Rice encoding.